README.md

```diff
@@ -92,6 +92,10 @@ Reference images can be found in [kubernetes-sigs/image-builder](https://github.
  prerequisites in the image, you can always deploy and execute some custom scripts 
  through the [KubeadmConfig](https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm).  
 
+Images built with [kubernetes-sigs/image-builder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi)
+based on Ubuntu are provided via [23technologies/kubernetes-capi-images](https://github.com/23technologies/kubernetes-capi-images).
+URLs to download the images can be found there in the [README](https://github.com/23technologies/kubernetes-capi-images#openstack) file.
+
```

Comment:
[23technologies/kubernetes-capi-images](https://github.com/23technologies/kubernetes-capi-images) is not "official" image. If you want "official" downloadable image, consult with [image-builder](https://github.com/kubernetes-sigs/image-builder) project, please.

Reply:
Is not intended to be an official image. We just have them available and I think a reference to it makes sense that there are no prebuilt images anywhere else.

Reply:
I think it would be good to add a link, but please with a disclaimer that these are not images officially maintainted by CAPO/ CAPI sig-cluster-lifecycle, just to avoid misunderstandings.

Reply:
Added "by the community for the community". Is this sufficient?

Reply:
I'm not sure this is clear enough, because it's not really clear what the difference between the "official" (kubernetes-sigs) CAPO/CAPI community and CAPO "user" community is.